### ISSUE TITLE
Decide whether the RIC covariance scale should be a config knob

### ISSUE THREAD

**Issue by chrisfinlay** [epfl-radio-astro/tabascal#224]
`satellites.ric_std` was removed in #222: it sat in all three shipped configs and was documented as a live parameter, and nothing read it.

## What is actually there

`FixedOrbit` and `Orbit` both hard-code the covariance (`trajectory.py:391`, `trajectory.py:573`):

```python
self.ric_cov = jnp.diag(jnp.array([0.73, 1.31, 0.54, 0.1, 0.1, 0.1])**2)/1e4
# self.ric_std = config.args["satellites"]["ric_std"]
```

The line that would read the config is commented out in both. The shipped configs all set `ric_std: 1e2`, and `1e2**2 == 1e4`, so the hard-coded divisor is exactly the shipped value squared — the knob is baked in at the value everything was setting anyway.

## Why it was not simply rewired

The relationship is not recoverable from the code. A divisor of `ric_std**2` and a multiplier of `(1/ric_std)**2` are the same arithmetic, and the docs described it as "a factor by which to scale the RIC covariance" while the code divides — so whether a larger `ric_std` should widen or narrow the prior is a modelling decision, not a transcription. Guessing it inside a stripping change would have been a silent change to how satellite positions are fitted.

## What to decide

- Whether the RIC covariance scale is a knob worth exposing at all, or whether the fixed value from the literature is the right answer for every observation.
- If it is exposed: the sense of the parameter (a std multiplier on the covariance, most likely, so `ric_cov = base * ric_std**2`) and a default that reproduces today's numbers.
- Where the base `[0.73, 1.31, 0.54, 0.1, 0.1, 0.1]` and the `/1e4` come from. The docs said "a paper where the average errors are calculated" without naming it, and that citation is worth having in the code.

Related: #222 removed the key; #101 covers the separate question of how record age bounds positional accuracy.
